wordpress-mobile/WordPress-Android
# Enable unread post counter feature.

This PR enables an unread post count feature.

To test:
- Install the app using vanilla flavor.
- Make sure you haven't manually overridden the flag.
- Confirm that the unread post count is visible in Reader's Following site filter.

[![Image from Gyazo](https://i.gyazo.com/aff0b40b6015af5d21cd05b65779a244.png)](https://gyazo.com/aff0b40b6015af5d21cd05b65779a244)

PR submission checklist:
